carsus/tests/data/vald_shortlist_test.dat

```diff
@@ -0,0 +1,100 @@
+ WARNING: Output was truncated to 100000 lines
+ 5000.00000, 5020.00000, 100000, 100001, 1.0 Wavelength region, lines selected, lines processed, Vmicro
+                                                 Damping parameters   Lande  Central
+Spec Ion       WL_air(A) Excit(eV) Vmic log gf*  Rad.   Stark  Waals  factor  depth  Reference
```

Comment:
a function that takes this `WL_air(A)` and transforms it to enum of air and vacuum (or bool) and then to unit